src/main/java/ArrayReverse.java

```diff
@@ -1,5 +1,24 @@
+import java.util.Arrays;
+
 public class ArrayReverse {
     public static void main (String[] args) {
-        System.out.println("hello world");
+        String[] arr1 = new String[4];
+        arr1[0] = "a";
+        arr1[1] = "b";
+        arr1[2] = "c";
+        arr1[3] = "d";
+        System.out.println(Arrays.toString(arr1));
+        arr1 = reverse(arr1);
```

Comment:
You called your function reverseArray. Leaving line 11 as arr1 = reverse(arr1) creates an error since there is no function called "reverse".